docset/winserver2022-ps/hyper-v/Dismount-VMHostAssignableDevice.md

```diff
@@ -0,0 +1,218 @@
+---
+description: Dismounts an assigned device from a virtual machine host.
+external help file: Microsoft.HyperV.PowerShell.Cmdlets.dll-Help.xml
+Module Name: Hyper-V
+ms.date: 06/12/2024
+online version: https://learn.microsoft.com/powershell/module/hyper-v/dismount-vmhostassignabledevice?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Dismount-VMHostAssignableDevice
+---
+
+# Dismount-VMHostAssignableDevice
+
+## SYNOPSIS
+Dismounts a device from a virtual machine (VM) host.
+
+## SYNTAX
+
+```
+Dismount-VMHostAssignableDevice [-InstancePath <String>] [-LocationPath <String>] [-Force]
+ [-Passthru] [-CimSession <CimSession[]>] [-ComputerName <String[]>] [-Credential <PSCredential[]>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Dismount-VMHostAssignableDevice` cmdlet is used to dismount an assignable device from the
+virtual machine host. Use this cmdlet when you need to reassign a device or resolve conflicts
+between devices and virtual machines.
+
+## EXAMPLES
+
+### Example 1
+
+```powershell
+Dismount-VMHostAssignableDevice -InstancePath "PCIROOT(0)#PCI(1D02)#PCI(0000)"
+```
+
+This example dismounts the device with the specified instance path from the host, making it
+available for reassignment to another virtual machine.
```

Comment:
Suggestion: "... making it available for **assignment to a virtual** machine."

The physical VM hosting machine is not a VM, so consumers might find it confusing to talk about reassignment